db-lb-app/infrastructure/openstack.yaml

```diff
@@ -135,15 +144,15 @@ node_templates:
       client_config: *client_config
       use_external_resource: true
       resource_config:
-        id: { get_capability: [ openstack, public_network ] }
+        id: { get_capability: [ openstack-network, public_network ] }
```

Comment:
Please don't change the name of this, if you want it to be differnt in the demo add and input `network_deployment_name` and use here { get_capapability: [ { get_input: network_deployment_name }, public_network ] }`

Reply:
done